pl_examples/basic_examples/conv_sequential_example.py

```diff
@@ -173,17 +170,16 @@ def configure_optimizers(self):
 
 
 def instantiate_datamodule(args):
-    train_transforms = torchvision.transforms.Compose([
-        torchvision.transforms.RandomCrop(32, padding=4),
-        torchvision.transforms.RandomHorizontalFlip(),
-        torchvision.transforms.ToTensor(),
-        cifar10_normalization(),
-    ])
-
-    test_transforms = torchvision.transforms.Compose([
-        torchvision.transforms.ToTensor(),
-        cifar10_normalization(),
-    ])
+    train_transforms = torchvision.transforms.Compose(
+        [
+            torchvision.transforms.RandomCrop(32, padding=4),
+            torchvision.transforms.RandomHorizontalFlip(),
+            torchvision.transforms.ToTensor(),
+            cifar10_normalization(),
+        ]
+    )
```

Comment:
the past collapsing was cuter :rabbit:

Reply:
I believe the reason for this is that so if you pass more arguments to `Compose`, your diff will impact less lines:

```python
result = a_long_function_name([
    "the red fox jumped",
    "over the fence",
    "foo bar bar",
])
```

later...
```python
result = a_long_function_name([
    "the red fox jumped",
    "over the fence",
    "foo bar foobar",
], "another_argument", "maybe_one_more")
```

Reply:
+1 to @carmocca's comment. this becomes very useful for preserving git blame